test/visibility/import/edgeCases/anotherMultipleDef.good

```diff
@@ -1 +1 @@
-type method A.foo() for class A
+Module scope A.foo()
```

Comment:
Making sure I'm keeping the rules straight:  This is not an ambiguity because `use B` introduces `A` in a shadow scope, but `import A;` makes `A` available directly in this scope.  Checking the OP, I'm pretty sure that's right.